dotnetv3/StepFunctions/README.md

```diff
@@ -0,0 +1,128 @@
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+<!--custom.important.start-->
+<!--custom.important.end-->
+
+## Code examples
+
+### Get started
+
+* [Hello Step Functions](Actions/HelloStepFunctions.cs#L4) (`ListStateMachines`)
+
+### Single actions
+
+Code excerpts that show you how to call individual service functions.
+
+* [Create a state machine](Actions/StepFunctionsWrapper.cs#L42) (`CreateStateMachine`)
+* [Create an activity](Actions/StepFunctionsWrapper.cs#L28) (`CreateActivity`)
+* [Delete a state machine](Actions/StepFunctionsWrapper.cs#L83) (`DeleteStateMachine`)
+* [Delete an activity](Actions/StepFunctionsWrapper.cs#L68) (`DeleteActivity`)
+* [Describe a state machine](Actions/StepFunctionsWrapper.cs#L114) (`DescribeStateMachine`)
+* [Describe a state machine run](Actions/StepFunctionsWrapper.cs#L99) (`DescribeExecution`)
+* [Get task data for an activity](Actions/StepFunctionsWrapper.cs#L129) (`GetActivityTask`)
+* [List activities](Actions/StepFunctionsWrapper.cs#L147) (`ListActivities`)
+* [List state machine runs](Actions/StepFunctionsWrapper.cs#L176) (`ListExecutions`)
+* [List state machines](Actions/StepFunctionsWrapper.cs#L206) (`ListStateMachines`)
+* [Send a success response to a task](Actions/StepFunctionsWrapper.cs#L227) (`SendTaskSuccess`)
+* [Start a state machine run](Actions/StepFunctionsWrapper.cs#L245) (`StartExecution`)
+
+### Scenarios
+
+Code examples that show you how to accomplish a specific task by calling multiple
+functions within the same service.
+
+* [Get started with state machines](Scenarios/UIMethods.cs) 
+
+## Run the examples
+
+### Prerequisites
+
+
+For prerequisites, see the [README](../README.md#Prerequisites) in the `dotnetv3` folder.
+
+
+
+<!--custom.prerequisites.start-->
+<!--custom.prerequisites.end-->
+
+### Instructions
+
+
+<!--custom.instructions.start-->
+<!--custom.instructions.end-->
+
+#### Hello Step Functions
+
+This example shows you how to get started using Step Functions.
+
+
+#### Get started with state machines
+
+This example shows you how to do the following:
+
+* Create an activity.
+* Create a state machine from an Amazon States Language definition that contains the previously created activity as a step.
+* Run the state machine and respond to the activity with user input.
+* Get the final status and output after the run completes.
+* Delete resources created by the example.
+
+Before you compile the .NET application, you can optionally set configuration values
+in the settings.json file. These settings include your account and topic settings for
```

Comment:
I agree with putting information in here on what configuration values they need to set, but these look like Cloudwatch settings instead of Step Functions. If it is a WriteMe problem, you can adjust them manually here.

Reply:
I adjusted it manually and will doe some research into how this happened.